.openshift-applier/apply.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
I’d recommend just using the playbook in the openshift-applier implementation - one less thing to maintain